tests/foreman/api/test_oscap_tailoringfiles.py

```diff
@@ -0,0 +1,69 @@
+# -*- encoding: utf-8 -*-
+"""Test class for Tailoring Files
+
+:Requirement: tailoringfiles
+
+:CaseAutomation: Automated
+
+:CaseLevel: Acceptance
+
+:CaseComponent: SCAPPlugin
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+from nailgun import entities
+
+from robottelo.config import settings
+from robottelo.datafactory import gen_string
+from robottelo.decorators import tier1
+from robottelo.helpers import file_downloader
+from robottelo.test import APITestCase
```

Comment:
The `APITestCase` still use the Unittest module, see https://github.com/SatelliteQE/robottelo/blob/8b778307a6f04374c40b78d972014da762b2f6f1/robottelo/test.py#L176